tests/_utils/test_prepare

```diff
@@ -32,7 +32,7 @@ function wait_pattern_exit() {
     pattern=$1
     while true
     do
-        if [ "pgrep -f $pattern" != "0" ]; then
+        if ! pgrep -f $pattern >/dev/null 2>&1; then
```

Comment:
Could you also fix the typo in L36 `echo "pattern $pattern already exist"`

Reply:
seems not a typo, it should be `process with pattern $pattern already exit`